src/routes/session/_Menu.svelte

```diff
@@ -68,6 +69,9 @@
 		on:mouseenter={() => {
 			loadVictorySound()
 		}}
+		on:click={() => {
+			quiz.updateDuration()
+		}}
```

Comment:
You can look at the first and last quizzed kana to determine the duration of the quiz instead of keeping this in sync. This should reduce the overall footprint while giving you the same result